benchmarks/benchmark_config_template.yaml

```diff
@@ -0,0 +1,34 @@
+# Torchserve version is to be installed. It can be one of the options
+#  - branch : "master"
+#  - nightly: "2022.3.16"
+#  - release: "0.5.3"
+# Nightly build will be installed if "ts_version" is not specifiged
+ts_version:
+    branch: &ts_version "master"
```

Comment:
From the documentation I would have guessed the syntax here should just be - same comment for hardware later down

`branch: "master"`